### ISSUE TITLE
Failed locale loading: Unexpected token 'const'.

### ISSUE THREAD

**Issue by AnoRebel** [nuxt-modules/i18n#2597]
### Environment

System:
    OS: Linux 5.0 undefined
    CPU: (8) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 0 Bytes / 0 Bytes
    Shell: 1.0 - /bin/jsh
  Binaries:
    Node: 18.18.0 - /usr/local/bin/node
    Yarn: 1.22.19 - /usr/local/bin/yarn
    npm: 9.4.2 - /usr/local/bin/npm
    pnpm: 8.10.5 - /usr/local/bin/pnpm

### Reproduction

[stackblitz.com/edit/nuxt-starter-jpctmx?file=package.json,i18n.config.ts,nuxt.config.ts](https://stackblitz.com/edit/nuxt-starter-jpctmx?file=package.json,i18n.config.ts,nuxt.config.ts)

### Describe the bug

Trying to setup a clean nuxt app with translations, using `@nuxtjs/i18n` and `@intlify/unplugin-vue-i18n` but the app fails with the error below:

### Screenshot

![image](https://github.com/intlify/bundle-tools/assets/20006734/35b30cec-c2d0-4803-8693-b8fd95538718)

### Expected behavior

No errors and translations to work normally.

### Additional context

Forwarded from https://github.com/intlify/bundle-tools/issues/324, since it's been 2 weeks with no response at all, I thought I might get some ideas from here.

### Logs

```shell
@nuxtjs/i18n Failed locale loading: [vite-node] [plugin:unplugin-vue-i18n] [VITE_ERROR] /locales/en.json <br><pre>const resource = {
  "title": (()=&gt;{const fn=(ctx) =&gt; {const { normalize: _normalize } = ctx;return _normalize(["Portfolio"])};fn.source="Portfolio";return fn;})()
}
export default resource</pre><br>

 ERROR  Internal server error: Unexpected token 'const'.                                                                                 3:08:59 PM
  Plugin: unplugin-vue-i18n
```

**Comment by BobbieGoede** [comment]
`@nuxtjs/i18n` uses `@intlify/unplugin-vue-i18n` internally so you don't have to configure it yourself. 

I'm not entirely ure what you're trying to do, I have changed your reproduction to demonstrate what it would look like if you don't configure `@intlify/unplugin-vue-i18n` yourself [here](https://stackblitz.com/edit/nuxt-starter-a7ln86?file=nuxt.config.ts).

**Comment by AnoRebel** [comment]
> `@nuxtjs/i18n` uses `@intlify/unplugin-vue-i18n` internally so you don't have to configure it yourself.

I see, I hadn't seen anywhere in the `nuxtjs/i18n` docs where they said they use `@intlify/unplugin-vue-i18n` internally so i followed the `@intlify/unplugin-vue-i18n` installation [docs](https://github.com/intlify/bundle-tools/tree/main/packages/unplugin-vue-i18n#-installation) as you can see from the link.

> I'm not entirely ure what you're trying to do, I have changed your reproduction to demonstrate what it would look like if you don't configure `@intlify/unplugin-vue-i18n` yourself [here](https://stackblitz.com/edit/nuxt-starter-a7ln86?file=nuxt.config.ts).

Does this also mean that I don't need to manually install `@intlify/unplugin-vue-i18n`? Because I need to access `@intlify/unplugin-vue-i18n/messages` to add to the `i18n.config.ts`.

So, basically from your reproduction, it means my error/problem comes from me trying to setup `@intlify/unplugin-vue-i18n` manually inside the vite config block in the `nuxt.config.ts`.

**Comment by BobbieGoede** [comment]
Oh I didn't notice the usage in `i18n.config.ts`, you don't have to setup `@intlify/unplugin-vue-i18n` there either. You already configured locale files in `nuxt.config.ts`, these will be loaded and used, Nuxt I18n will pass the messages to Vue I18n internally. If you're using `<i18n>` blocks in components, these will also be automatically picked up and used.

You probably don't need to install `@intlify/unplugin-vue-i18n` at all, unless there are use cases for this that I am not aware of, most typical usage should be covered without additional configuration.

**Comment by kazupon** [comment]
Thank you your reporting!

As @BobbieGoede mentioned, `@intlify/unplugin-vue-i18n` is already installed, so you don't need to install it with nuxt.config.
please remove it from nuxt.config and see if it works. 🙏

**Comment by AnoRebel** [comment]
> Oh I didn't notice the usage in `i18n.config.ts`, you don't have to setup `@intlify/unplugin-vue-i18n` there either. You already configured locale files in `nuxt.config.ts`, these will be loaded and used, Nuxt I18n will pass the messages to Vue I18n internally. If you're using `<i18n>` blocks in components, these will also be automatically picked up and used.
> 
> You probably don't need to install `@intlify/unplugin-vue-i18n` at all, unless there are use cases for this that I am not aware of, most typical usage should be covered without additional configuration.

Okay, thank you for the help and quick response.

**Comment by AnoRebel** [comment]
> Thank you your reporting!
> 
> As @BobbieGoede mentioned, `@intlify/unplugin-vue-i18n` is already installed, so you don't need to install it with nuxt.config. please remove it from nuxt.config and see if it works. 🙏

I have removed the vite config block from nuxt.config and uninstalled the manually installed `@intlify/unplugin-vue-i18n`. Things seem to be working fine.
Again, thank you for the help. I'll close this issue with this comment, unless I mess something up again and cant fix it on my own. :smile:
